packages/urlstate/encoder/encoder.ts

```
@@ -14,27 +14,50 @@ export function encode(payload: unknown): string {
+
+export const encodePrimitive = (payload: Primitive) => {
   switch (typeOf(payload)) {
     case "function":
     case "symbol":
       return "";
     case "date":
-      return encodeDate(payload as Date);
+      return SYMBOLS.date + new Date(payload as Date).toISOString();
     case "undefined":
       return SYMBOLS.undefined;
+    case "number":
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)